VultisigApp/VultisigApp/Services/CoinService/VaultDefaultCoinService.swift

```diff
@@ -8,7 +8,7 @@
 import Foundation
 import SwiftData
 
-
+// Should I move this to Vault service?
```

Comment:
is it still a question?

Reply:
I'm still thinking about this one. What is your suggestion?